eva_cttv_pipeline/evidence_string_generation/evidence_strings.py

```diff
@@ -296,6 +286,14 @@ def date(self, date_string):
         self['evidence']['gene2variant']['date_asserted'] = date_string
         self['evidence']['variant2disease']['date_asserted'] = date_string
 
+    @property
+    def last_evaluated_date(self):
+        return self['evidence']['variant2disease']['last_evaluated_date']
```

Comment:
I'm only thinking of this now and it would apply to other PRs. But i would be safe to have a `.get('last_evaluated_date')`  as it would avoid raising a  `KeyError` if it is called before being set.

Reply:
The way evidence strings classes work now, the getters are never supposed to be called before all of the properties have been populated. And the problem I see with using `.get('last_evaluated_date')` is that, if due to some coding mistake that getter _would_ be called before the field is populated, it would ignore the issue and silently return None. I would prefer it raising a `KeyError` in this case to indicate to me that something is broken.

Reply:
Sounds good